### ISSUE TITLE
Welcome to Social Winter Of Code 2023

### ISSUE THREAD

**Issue by Sulagna-Dutta-Roy** [WeBeginners-Community/CSS.Effects-Designs#302]
## Check the guidelines first

**Comment by AnkitMajee** [comment]
**From when we will raise our issue**

**Comment by Sulagna-Dutta-Roy** [comment]
Kindly check the guidelines from discord for now. We admins do not have any infos

**Comment by AnkitMajee** [comment]
Yeah I got SWOC discord mail twice but both the links are Invalid , if you the discord link have please share it here

**Comment by Sulagna-Dutta-Roy** [comment]
Sorry, it's a open platform so I can't share the discord link here. You can send a mail to swoc and then we will send you the link.

**Comment by AnkitMajee** [comment]
ok

**Comment by muskaanrajput14** [comment]
When will the issues be raised??

**Comment by SattyamSamania** [comment]
Hey @Sulagna-Dutta-Roy Can we add JS projects also?

**Comment by goyalh4164** [comment]
I would like to contribute a Landing Page (Fully Responsive) using Bootstrap? Can we contribute that?
